lib/serverless-tools/deployer/docker_builder.rb

```diff
@@ -8,7 +8,7 @@ def initialize(config:)
       end
 
       def build
-        system("docker build . -f #{config.dockerfile} -t #{local_image_name}")
+        system("docker build . -f #{config.dockerfile} -t #{local_image_name}#{platform}")
```

Comment:
Super tiny nitpick, but might it be better to put the whitespace here (before `#{platform}`, rather than in the `platform` method, so that it doesn't look like something connected to the `#{local_image_name}`?

Reply:
Good shout - I've done that now @elipinska